doc/restapi/introduction.rst

```diff
@@ -18,9 +18,9 @@ Some requests require one or more parameters. These parameters are passed using
 
     curl -X PUT -H "X-Api-Key: <YOUR API KEY>" http://localhost:20100/mychannel/rssfeeds/http%3A%2F%2Frssfeed.com%2Frss.xml
```

Comment:
There is a reference to `20100` here. To maintain consistency, it would be good to update this as well.

Reply:
Thanks, I changed it to `<port>`